tree/treeplayer/test/array.cxx

```diff
@@ -66,3 +67,138 @@ TEST(TTreeReaderArray, MultiReaders) {
       EXPECT_DOUBLE_EQ(Double[i], trDouble5[i]);
    }
 }
+
+TEST(TTreeReaderArray, Iterator) {
```

Comment:
Sweeet! Thanks! Should this be moved into iterval.cxx? I'd say it has acquired critical mass...

Reply:
As you prefer, but I'd opt for a more verbose name...would `readerarray_iterator.cxx` be ok?